projects/plugins/boost/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.1-beta] - 2022-03-08
+### Added
+- Added internal support for Cloud Critical CSS.
+- Added My Jetpack page.
+- Added suggested steps when Critical CSS fails due to page errors.
+
+### Changed
+- Critical CSS: Exclude animation keyframes from Critical CSS Generation.
+- Deferred JS: Attribute to exclude scripts from deferral now allows quotes.
+- General: Boost is now compatible with WordPress 5.9.
+- Tidy up Critical CSS class structure.
+- Updated Dashboard heading with new Horizontal logo.
```

Comment:
There is a bit of a mix of present and past tense in here, and some entries with a prefix (General, Critical CSS, Deferred JS) while others do not have one. Maybe we could standardize this?

Reply:
Fixed in f5f3d43603.